examples/FORTRAN/H5D/Fortran_sourcefiles.cmake

```diff
@@ -0,0 +1,24 @@
+#-----------------------------------------------------------------------------
+# Define Sources, one file per application
+#-----------------------------------------------------------------------------
+set (examples
```

Comment:
All these will work across HDF5 versions?

Reply:
I believe so, but I have not got Fortran to work CMake.

Reply:
CMake Error at /usr/share/cmake/Modules/CheckFortranSourceCompiles.cmake:100 (message):
  Unknown argument:

    SRC_EXT.F90

Call Stack (most recent call first):
  examples/config/cmake/HDFUseFortran.cmake:113 (check_fortran_source_compiles)
  examples/CMakeLists.txt:119 (include)


-- Configuring incomplete, errors occurred!

Reply:
Why did you add the HDFUseFortran.cmake file?